react-components/src/architecture/concrete/pointsOfInterest/PointsOfInterestDomainObject.ts

```diff
@@ -36,7 +42,7 @@ export class PointsOfInterestDomainObject<PoiIdType> extends VisualDomainObject
         ...this._pointsOfInterest
       ];
       this.notify(Changes.geometry);
-    });
+    }); */
```

Comment:
this commented code should be removed or is it intentional to keep it for now and enable it later?
If is that the case, then I would suggest to add a comment as a header to explain when it should be enabled again.

Reply:
Woops :see_no_evil: I'll remove